src/v/redpanda/admin/api-doc/migration.def.json

```diff
@@ -14,6 +14,37 @@
         }
     }
 },
+"labeled_namespaced_topic": {
+    "type": "object",
+    "required": [
+        "topic"
+    ],
+    "properties": {
+        "label": {
```

Comment:
do we call this label or cluster_uuid? @bashtanov @andrwng

Reply:
I originally named it label in case we ever wanted to add more stuff to it (e.g. user defined per-cluster tag). Ultimately the end user will only care about how this will be used, which is as a hint for finding the correct topic manifest, so maybe let's keep it generic with "label"?

For reference here's where I think a human will use this label (first listing the mount manifests, and then passing the label as a hint, though @bashtanov please correct me if this sounds off) https://github.com/redpanda-data/redpanda/blob/d8ac9b0657ceb829ee9e03b44e13dc6f1604df38/src/v/cluster/data_migration_backend.cc#L619-L622

Reply:
I'd say the _lack_ of human interface is here
https://github.com/redpanda-data/redpanda/blob/b346d8d480b5f8b095d2f29ec770bb1da90de8e0/src/v/redpanda/admin/data_migration_utils.cc#L26-L33

So supporting it is really a matter of API and testing, the rest is there.